server/templates/menu_dev.html

```diff
@@ -0,0 +1,47 @@
+<!--
```

Comment:
Nit: Please use jinja comments {# ... #}
That won't render with the page load.

Reply:
got it.